cmake-modules/AzureDoxygen.cmake

```diff
@@ -19,7 +19,15 @@ function(generate_documentation PROJECT_NAME PROJECT_VERSION)
         set(DOXYGEN_RECURSIVE YES)
         set(DOXYGEN_USE_MDFILE_AS_MAINPAGE ./README.md)
         set(DOXYGEN_INLINE_SOURCES YES)
+        # Skip generating docs for json, test and private files.
+        set(DOXYGEN_EXCLUDE_PATTERNS
+            json.hpp
+            test
```

Comment:
Does this need to be `*/test/*` to catch nested directories, or will a simple string like this work as expected?

Reply:
work as expected